src/Management/test/Endpoint.Test/ActuatorsHostBuilderTest.cs

```diff
@@ -165,7 +165,9 @@ public async Task HealthActuatorWithDetails(HostBuilderType hostBuilderType)
     {
         var appSettings = new Dictionary<string, string?>(AppSettings)
         {
-            { "Management:Endpoints:Health:ShowDetails", "Always" }
+            ["Management:Endpoints:Health:ShowComponents"] = "Always",
```

Comment:
Based on the name of this test, it should also set `ShowDetails` to `Always`.

Reply:
Updating test name to match original intent